### DISCUSSION TITLE
Possible silent conversion of u64 to f64

### DISCUSSION THREAD

**Discussion by mike-ward** [vlang/v#21770] [Questions and Answers] [answered]
```
mut num := f64(0.0)
num = strconv.atof64(some_string) or { strconv.double_minus_infinity }
```
`strconv.double_minus_infinity` is `u64`. This compiles OK but I feel it should be an error because a `u64` is converted to a `f64 ` in the or clause

**Comment by medvednikov** [comment, marked as answer]
You're right, this is a bug.

Good find. Please report it via github issues.
